pkg/controller/performanceprofile/components/kubeletconfig/kubeletconfig_test.go

```diff
@@ -25,7 +25,7 @@ var _ = Describe("Kubelet Config", func() {
 		selectorKey, selectorValue := components.GetFirstKeyAndValue(profile.Spec.MachineConfigPoolSelector)
 		Expect(manifest).To(ContainSubstring(fmt.Sprintf("%s: %s", selectorKey, selectorValue)))
 		Expect(manifest).To(ContainSubstring("reservedSystemCPUs: 0-3"))
-		Expect(manifest).To(ContainSubstring("topologyManagerPolicy: best-effort"))
+		Expect(manifest).To(ContainSubstring("topologyManagerPolicy: single-numa-node"))
```

Comment:
I think to need to set it here: https://github.com/openshift-kni/performance-addon-operators/blob/master/pkg/utils/testing/testing.go#L30

Reply:
I've added some logic to testing functions.